### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nate-goudreault**
Adding resource locations to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ This repository contains Python notebooks and R Markdown guides for achieving sp
 
 Start learning with the [API Training](https://github.com/datarobot-community/tutorials-for-data-scientists/tree/master/DRU/API_Training) module.
 
+Additional code examples can be accessed from the following locations:
+
+- [Tutorials for data scientists repo](https://github.com/datarobot-community/tutorials-for-data-scientists/)
```
Comment (lindahaviland) [inline README.md:9]: this is also listed under "Important Links" - guessing you should remove that other link

File: README.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ This repository contains Python notebooks and R Markdown guides for achieving sp
...
+- [AI Engineering repo](https://github.com/datarobot-community/ai_engineering)
+- [docs.datarobot.com](https://docs.datarobot.com/en/docs/api/code-examples/index.html)
+- [R Vignettes included in the R client](https://cran.r-project.org/web/packages/datarobot/index.html)
+- [Examples included in the Python client](https://datarobot-public-api-client.readthedocs-hosted.com/en/v2.27.1/examples/index.html)
+
```
Comment (lindahaviland) [inline README.md:14]: Additional changes under Important Links:

- To learn how to use DataRobot and gain practical data science education, visit [DataRobot University](https://university.datarobot.com/).
-  To ask questions and get help from DataRobot experts and peers, visit [DataRobot Community](https://community.datarobot.com/t5/datarobot-community/ct-p/en).

**Review by lindahaviland — CHANGES_REQUESTED** [review]
couple of small things. Guessing you want to remove the link to the repo since it's now in your new list. thank you!

**Comment by nate-goudreault** [comment]
@lindahaviland this and the other two PRs are fixed and ready for review

https://github.com/datarobot-community/ai_engineering/pull/43

https://github.com/datarobot-community/tutorials-for-data-scientists/pull/9

**Review by lindahaviland — APPROVED** [review]
Looks good, thank you.
 Future: Just noticed we'll want to change the https://github.com/datarobot-community/examples-for-data-scientists/blob/0ab50428ae2822cd5732cd8b838d57bbcb93166f/CONTRIBUTING.md and https://github.com/datarobot-community/examples-for-data-scientists/blob/0ab50428ae2822cd5732cd8b838d57bbcb93166f/CODE_OF_CONDUCT.md pages, but not part of this ticket. These are linked from all readmes (or almost all) and are out-of-date.

**Pull request merged**